sbt-test/source-dependencies/stable-ctx-bounds/test

```diff
@@ -0,0 +1,9 @@
+> compile
```

Comment:
may be worth noting here the predicted order of typer visiting definitions in run 1 vs run 2, to illustrate how before the change it actually had unstable names between runs.